app/src/main/java/grinnell/appdev/edu/lyles/MenuViewPagerAdapter.java

```diff
@@ -18,11 +16,11 @@
  * Created by Mattori on 9/25/16.
  */
 public class MenuViewPagerAdapter extends FragmentStatePagerAdapter {
-    private static final int HOT_FOOD_INDEX = 0;
-    private static final int SNACKS_INDEX = 1;
-    private static final int DRINKS_INDEX = 2;
-    private static final int FAVORITES_INDEX = 3;
-    private static final int NUM_TABS = 4;
+    public static final int HOT_FOOD_INDEX = 0;
```

Comment:
These can just be moved to `MenuTabFragment` and made private, since they aren't used anywhere else.